package-drone/src/main/java/de/dentrassi/pm/jenkins/DroneRecorder.java

```diff
@@ -485,19 +433,40 @@ public Void invoke ( final File basedir, final VirtualChannel channel ) throws I
                             filename = f;
                         }
                         uploader.upload ( file, filename );
-//                        TODO this.artifacts.put ( artifacId, f );
                     }
-
+                    uploadResult.addArtifacts ( uploader.complete () );
```

Comment:
What about if in V2 upload fails on thrird file? The upload result contains the uploaded files with success?